src/ARToolkit.Forms/ARSceneView.cs

```diff
@@ -36,6 +36,25 @@ public ARSceneView()
             OriginCamera = new Mapping.Camera(0, 0, 15E6, 0, 0, 0);
         }
 
+        /// <summary>
+        /// Identifies the <see cref="ClippingDistance"/> bindable property.
+        /// </summary>
+        public static readonly BindableProperty ClippingDistanceProperty =
+            BindableProperty.Create(nameof(ClippingDistance), typeof(double), typeof(ARSceneView), 0.0, BindingMode.OneWay, null);
```

Comment:
Confirming: Is the default 0.0 and not NaN or something else?

Reply:
Default is 0, and zero should mean no clipping